packages/components/app/styles/components/dropdown.scss

```
@@ -169,7 +170,22 @@ $hds-dropdown-toggle-border-width: 1px;
     isolation: isolate; // used to create a new stacking context (needed to have the pseudo element below text/icon but not the parent container)
     min-height: 36px;
 
-    a {
+    // need to reset a few extra things to make the button visually appear the same as the links
+    // also CR if you want to re-factor, that's fine; just trying to make this work in this first pass
+    // TODO this is 2px taller than the link...why?
+    button {
+      box-sizing: border-box;
```

```suggestion
      box-sizing: border-box;
```
all these `box-sizing: border-box` declaration can be removed now (after merging `main`)